apps/files/src/store/actions.js

```diff
@@ -399,7 +400,20 @@ export default {
         .finally(() => {
           context.commit('UPDATE_FOLDER_LOADING', false)
           client.users.getUser(context.rootGetters.user.id).then(res => {
-            context.commit('CHECK_QUOTA', res.quota)
+            const quota = res.quota
+
+            context.commit('CHECK_QUOTA', quota)
+
+            // Display quota in the sidebar
+            if (
+              !isPublicPage &&
+              !context.getters.currentFolder.isMounted() &&
+              quota.definition !== 'default' &&
+              quota.definition !== 'none'
+            ) {
+              context.commit('SET_SIDEBAR_FOOTER_CONTENT_COMPONENT', SidebarQuota, { root: true })
```

Comment:
is there a code path where maybe the quota might have changed and we need to set an empty Vue into that extension point ? I could imagine this happening if the user is switching between views/apps while the admin might have changed the quota from a true value to infinite. it's quite a corner case though

Reply:
Right - I added an `else` which will reset the component in there.